src/components/core/Header.vue

```diff
@@ -177,4 +180,7 @@ q-toolbar.row.q-py-sm.q-px-md.bg-grey-1
   align: left
 .blue
   color: $primary
+.btn
```

Comment:
- You can remove this class and its references
- The right attribute can be replaced with `q-mr-xs` or similar on the q-icon
- The cursor pointer is not necessary for mobile

Reply:
Hello Jacques, duly noted. I will make the changes.